docs/coding.rst

```diff
@@ -291,28 +291,6 @@ Release note templates live here: https://github.com/mozilla/bedrock/tree/main/b
 - Firefox iOS: https://www.mozilla.org/en-US/firefox/ios/99.0/releasenotes/
 
 
-Optimizing Images
------------------
-
-Images can take a long time to load and eat up a lot of bandwidth. Always take care
-to optimize images before uploading them to the site.
-
-The script ``img.sh`` can be used to optimize images locally on the command line:
-
```

Comment:
_(It's sad that it has to go since tinypng has the best algos I've ever tried…)_

I think that the section itself could probably stay, reminding authors to optimise PNGs and JPGs manually (either locally like imageoptim, or online through tinypng web interface), and cleaning up SVGs with svgo when adding/editing assets to get rid of funky markup, macros et al.

I'd love to propose any alternative, maybe like a hook or action to take care of it all, but honestly never found a good solution to this myself :(

Reply:
+1 on keeping this section available in the docs to remind developers/contributors to optimize images for performance 👍🏼 

We could also just add the actual [tinypng.com](https://tinypng.com/) website as reference here if we'd like to keep mention of it :) I usually use that page or https://compresspng.com/ anyway!

Reply:
Good suggestion, thanks both!